app/Config/Users.php

```diff
@@ -24,7 +24,7 @@ class Users extends BaseConfig
      * The default image type to use from Gravatar if they do not have an
      * image for that user.
      */
-    public $gravatarDefault = 'blank';
+    public $gravatarDefault = 'mp';
```

Comment:
Doesn't this cover up their initials when they don't have an avatar?

Reply:
I missed the `renderAvatar` method and related view. Now I understand why we were using the `blank` option. 

However, we won't display initials since the gravatar always generates a link when gravatar is enabled: https://github.com/lonnieezell/forum-example/blob/develop/app/Views/users/_avatar.php#L4